main.js

```diff
@@ -1323,6 +1340,10 @@ require(['jquery', 'underscore', 'utils', 'video', 'soundchip', 'ddnoise', 'debu
                     dbgr.debug(processor.pc);
                     throw e;
                 }
+                if (stepEmuWhenPaused) {
```

Comment:
All looks good, maybe a better name though? Naming is hard...I know...but "stop emu when paused" confuses me: what is pausing? what's a step? Would 'singleStepFrame' make more sense? We can probably make a single keypress (shift G maybe?) in the regular debugger do this, instead of having another mode: but this works too :) I want jsbeeb to be useful to your debugging style, and I'm not 100% sure of the use case :)

Reply:
Ahh (leaving original comment in place!)

I see: pausing is a new state that doesn't drop into the debugger!

Looks good to me !